test/unit-tests.test.ts

```diff
@@ -2,6 +2,7 @@ import { compile } from '../ts/index';
 import { sandbox } from '../ts/sandbox';
 import * as runtime from '../ts/runtime';
 import { default as generator } from 'babel-generator';
+import 'jest'
```

Comment:
Why is this required?

Reply:
Oh sorry, I accidentally left it there. I'll remove it